scripts/release/mule/test/test.sh

```diff
@@ -9,13 +9,25 @@ ARCH_TYPE=$(./scripts/archtype.sh)
 export ARCH_TYPE
 OS_TYPE=$(./scripts/ostype.sh)
 export OS_TYPE
-VERSION=${VERSION:-$(./scripts/compute_build_number.sh -f)}
+
+if [ -z "$VERSION" ]; then
```

Comment:
Why is this additional check required?

Reply:
Good question.  This allows the env vars to be specified in the mule file, which comes in really handy when testing locally.

```
agents:
  - name: deb 
    dockerFilePath: docker/build/mule.go.debian.Dockerfile
    image: algorand/mule-linux-debian
    version: scripts/configure_dev-deps.sh
    buildArgs:
      - GOLANG_VERSION=`./scripts/get_golang_version.sh`
    env:
      - AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
      - AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
      - VERSION=1.2.3
      - BRANCH=foo
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
    workDir: $HOME/projects/go-algorand
```